project/Dependencies.scala

```diff
@@ -24,36 +24,36 @@ object Dependencies {
 
   val scalaJava8Compat = Seq("org.scala-lang.modules" %% "scala-java8-compat" % "0.9.0")
 
-  val playJsonVersion = "2.8.0-M1"
+  val playJsonVersion = "2.8.0-M3"
   val playJson = Seq("com.typesafe.play" %% "play-json" % playJsonVersion)
 
   val slf4jApi = Seq("org.slf4j" % "slf4j-api" % "1.7.26")
 
   val javaxInject = Seq("javax.inject" % "javax.inject" % "1")
 
-  val sslConfigVersion = "0.4.0"
+  val sslConfigVersion = "0.3.8"
   val sslConfigCore = Seq("com.typesafe" %% "ssl-config-core" % sslConfigVersion)
 
-  val scalaXmlVersion = "1.2.0"
-  val scalaXml = Seq("org.scala-lang.modules" %% "scala-xml" % scalaXmlVersion)
+  val scalaXmlVersion = "1.1.1"
+  val scalaXmlVersion213 = "1.2.0"
+  def scalaXml(scalaVersion: String) = if(scalaVersion.startsWith("2.13")) {
+    Seq("org.scala-lang.modules" %% "scala-xml" % scalaXmlVersion213)
+  } else {
+    Seq("org.scala-lang.modules" %% "scala-xml" % scalaXmlVersion)
+  }
 
   val signpostVersion = "1.2.1.2"
   val oauth = Seq("oauth.signpost" % "signpost-core" % signpostVersion)
 
-  val cachecontrolVersion = "2.0.0-M1"
+  val cachecontrolVersion = "2.0.0-M2"
   val cachecontrol = Seq("com.typesafe.play" %% "cachecontrol" % cachecontrolVersion)
 
   val asyncHttpClientVersion = "2.10.0"
   val asyncHttpClient = Seq("org.asynchttpclient" % "async-http-client" % asyncHttpClientVersion)
 
-  val akkaVersion = "2.6.0-M2"
+  val akkaVersion = "2.6.0-M3"
```

Comment:
in order to have Scala 2.13 support, we must bump the Akka version as well. 
Akka 2.6.0-M2 has on jars for Scala 2.13